libs/chain-signatures/contract/tests/expired_attestation.rs

```diff
@@ -81,168 +21,107 @@ impl TestSetup {
-        expiry_time_stamp_seconds: Some(EXPIRY_SECONDS),
-    });
-    let third_participant = setup.participants_list[2].0.clone();
-    setup.submit_attestation_for_participant(&third_participant, expiring_attestation);
-
-    assert_eq!(setup.contract.get_tee_accounts().len(), PARTICIPANT_COUNT);
-
-    // Fast-forward time past expiry and trigger resharing
-    const EXPIRED_TIMESTAMP: u64 =
-        INITIAL_TIMESTAMP_NANOS + Duration::from_secs(POST_EXPIRY_WAIT_SECONDS).as_nanos() as u64;
-    testing_env!(VMContextBuilder::new()
-        .block_timestamp(EXPIRED_TIMESTAMP)
-        .build());
-
-    assert!(!setup.contract.verify_tee().unwrap());
-
-    let resharing_state = match setup.contract.state() {
-        ProtocolContractState::Resharing(r) => r,
-        state => panic!("Should be in Resharing state. Actual state {:#?}", state),
-    };
-
-    // Complete resharing process
-    let key_event_id = KeyEventId::new(
-        resharing_state.prospective_epoch_id(),
-        domain_id,
-        AttemptId::new(),
-    );
-
-    testing_env!(create_context_for_participant(&participant_accounts[0]));
-    setup.contract.start_reshare_instance(key_event_id).unwrap();
-
-    // Vote for resharing with first 2 participants
-    for account_id in &participant_accounts {
-        testing_env!(create_context_for_participant(account_id));
-        setup.contract.vote_reshared(key_event_id).unwrap();
-    }
+    let now = worker
+        .status()
+        .await
+        .unwrap()
+        .sync_info
+        .latest_block_time
+        .unix_timestamp();
+    println!("now: {}", now);
+    let expiry: i64 = worker
+        .status()
+        .await
+        .unwrap()
+        .sync_info
+        .latest_block_time
+        .unix_timestamp()
+        + EXPIRY_OFFSET_SECONDS as i64;
 
-    // Verify final state: back to Running with one less participant
-    assert_matches!(setup.contract.state(), ProtocolContractState::Running(_));
-
-    // At this point we have 2 participants in Running state but 3 TEE accounts
-    // The cleanup step is tested separately in test_clean_tee_status_removes_non_participants
-    let final_running_state = match setup.contract.state() {
-        ProtocolContractState::Running(r) => r,
-        _ => panic!("Should be in Running state after resharing"),
-    };
-
-    const EXPECTED_PARTICIPANT_COUNT: usize = PARTICIPANT_COUNT - 1;
-    assert_eq!(
-        final_running_state.parameters.participants().len(),
-        EXPECTED_PARTICIPANT_COUNT
-    );
-    // Before clean_tee_status() cleanup, we still have old TEE accounts
-    assert_eq!(setup.contract.get_tee_accounts().len(), PARTICIPANT_COUNT);
-}
-
-/// **Unit test for TEE cleanup of non-participants** - Tests that `clean_tee_status()` removes
-/// TEE accounts for accounts that are no longer in the participant list.
-/// This simulates cleanup after participant removal (e.g., post-resharing).
-#[test]
-fn test_clean_tee_status_removes_non_participants() {
```

Comment:
Can be removed, this is almost a duplicate of the one in `tee.rs`
https://github.com/near/mpc/blob/95649d0982a98942d6249e4d7efaf0758d0155b4/libs/chain-signatures/contract/tests/tee.rs#L335

Reply:
Discussed offline: https://nearone.slack.com/archives/C0912BTG51T/p1758033418475589.